codecov.yml

```diff
@@ -0,0 +1,2 @@
+fixes:
+  - "main/::"
```

Comment:
How come this path fix was needed?

Reply:
It's due to the `build` workflow, we clone the repo into a folder called `main` (see line 45). I guess could also have fixed there! But this seemed to work